features/cli/assume_role.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Using a stored role
+
+  In order to use a role
+  As an AWS IAM user
+  I want temporary security credentials to be generated for the role
+  And the credentials to be copied to ~/.aws/credentials
+
+  Given a user has already assumed a role
+  When the user wants to assume a new role
+  Then the user must assume a credential
+  And then assume the new role
+  This workflow could be improved by assuming the user wants to use their last
+  assumed credentials to assume the new role. Alternatively, we could introduce
+  the concept of "presets" which pairs a credential with a role and then the
+  user could assume the preset instead. The preset could check if an
+  unexpired session token already exists for the credential and automatically
+  use it or issue a new token if not.
```

Comment:
@zl4bv @tjnicholas I have just tested, and confirmed you can hop from one role to another, assuming the roles have appropriate trust documents, and the sts:assumerole permission has been granted.
I don't know how this effects the session timeout, though it appears that it is one hour from now.